src/prefabs/dataContainer.js

```diff
@@ -2,6 +2,90 @@
   name: 'DataContainer',
   icon: 'DataContainer',
   category: 'DATA',
+  beforeCreate: ({
+    prefab,
+    save,
+    close,
+    components: { ModelSelector, Header, Content, Field, Footer, Text },
+    helpers: { useCurrentPageId, camelToSnakeCase },
+  }) => {
+    const [modelId, setModelId] = React.useState('');
+    const [model, setModel] = React.useState(null);
+    const [validation, setValidation] = React.useState('');
+    const pageUuid = useCurrentPageId();
+
+    React.useEffect(() => {
+      setValidation('');
+    }, [modelId]);
+
+    return (
+      <>
+        <Header onClose={close} title="Configure dataContainer" />
```

Comment:
This should be: Configure data container

Reply:
Agreed, added to feedback list. Will resolve this along with the other feedback PO's have.